src/main/java/com/conveyal/gtfs/validator/MissingStopCoordinatesValidator.java

```diff
@@ -16,12 +16,11 @@ public boolean validate(GTFSFeed feed, boolean repair) {
         boolean isValid = true;
         for (Stop stop : feed.stops.values()) {
             Coordinate stopCoord = new Coordinate(stop.stop_lat, stop.stop_lon);
-
-            ProjectedCoordinate projectedStopCoord = null;
             try {
-                projectedStopCoord = GeoUtils.convertLatLonToEuclidean(stopCoord);
+                // attempt to project coordinate
+                GeoUtils.convertLatLonToEuclidean(stopCoord);
```

Comment:
@mattwigway, Actually, I think we can just get rid of this validator, no?  The existence of valid lat/lon values is already validated on load.